tfplan2cai/tfplan2cai.go

```diff
@@ -31,9 +32,9 @@ type TFPlanToCAIOptions struct {
 	AncestryCache map[string]string
 }
 
-type TFPlanToCAIFunc func(ctx context.Context, jsonPlan []byte, o *TFPlanToCAIOptions) ([]Asset, error)
+type TFPlanToCAIFunc func(ctx context.Context, jsonPlan []byte, o *TFPlanToCAIOptions) ([]caiasset.Asset, error)
```

Comment:
```suggestion
type ConvertFunc func(ctx context.Context, jsonPlan []byte, o *TFPlanToCAIOptions) ([]caiasset.Asset, error)
```

Reply:
I removed the ConvertFunc, realized it was not used anywhere.